Documentation/contributing/development/reviewers_committers/review_docs.rst

```diff
@@ -0,0 +1,228 @@
+process for Committers <review_process>` applies, even though it is not
+specific to documentation.
+
+Technical contents
+------------------
+
+You are not expected to review the technical aspects of the documentation
+changes in a PR. Obviously, if you do have knowledge of the topic and if you
+find some elements that are incorrect or missing, do flag them.
+
+Documentation structure
+-----------------------
+
+One essential part of the review is to ensure that the contribution maintains a
+coherent structure for the documentation. Ask yourself if the changes are
+located on the right page, at the right place. This is especially important if
+pages are added, removed, or shuffled around. If the addition is large,
+consider whether the page should be split. Consider also whether new text comes
+with a satisfactory structure: does it fit well with the surrounding context,
+or did the author simply use a "note" box instead of trying to integrate the
+new information to the relevant paragraph?
+
+See also :ref:`the recommendations on documentation structure for contributors
+<docs_structure_recommendations>`.
+
+Specific items to look out for
+------------------------------
+
+Backports labels
+~~~~~~~~~~~~~~~~
+
+See :ref:`the backport criteria for documentation changes
+<backport_criteria_docs>`. Mark the PR for backports by setting the labels for
+all supported branches to which the changes apply (all supported branches
+containing the parent features to which the modified sections relate).
+
+CODEOWNERS updates
+~~~~~~~~~~~~~~~~~~
+
+All documentation sources are assigned for review to cilium/docs-structure by
+default. However, when a contributor creates a new page, consider whether it
+should be covered by another team as well (for the technical aspects). If this
+is the case, ask the author to update the CODEOWNERS file.
+
+Beta disclaimer
+~~~~~~~~~~~~~~~
+
+When a feature is advertised as Beta in the PR, make sure that the author
+clearly indicates it as such in the documentation, both by mentioning "(Beta)"
+in the title of the section for the feature and by including the dedicated
+banner.
+
+Upgrade notes
+~~~~~~~~~~~~~
+
+When the PR introduces new user-facing options, metrics, or behavior, in such a
+way that it affects upgrades or downgrades, ensure that the author summarizes
+the changes with a note in Documentation/operations/upgrade.rst.
+
+Completeness
+~~~~~~~~~~~~
+
+Make sure that new or updated content is complete, with no TODOs.
+
+Auto-generated reference documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are some reference documents (command reference, Helm reference) that
+should be automatically updated when parts of the Cilium repository change. You
+should ensure that everything is up-to-date, although the CI does in fact
+handle the verification for you. But you may have to accompany contributors and
+help them figure out the command they need to run to perform the update.
```

Comment:
There are a couple of items in this paragraph that we discuss but don't provide specific definitions or pointers for. I think a user may have trouble understanding what exactly is meant here. For instance, I'm left with the following questions:

* What is a reference document used for?
* How can I identify a reference document?
* What changes to the Cilium repository require my attention in this matter?
* What do I need to do to ensure everything is up to date?
* How do I know what command I need to run to perform the update? Where can I find it?

Reply:
This would make for an entire separate guide, and I'm sure we'd struggle to keep it up-to-date :confused:. Not sure how to address this.

Reply:
Maybe we could add it as a todo and just leave this be for now?

Reply:
Yes, I'll open an issue for the build framework and mention this as well.